interp/interp_test.go

```diff
@@ -983,6 +991,25 @@ BEGIN { x[1]=3; f5(x); print x[1] }
 	{`{ print $1<$2 }`, "1_0 2", "1\n", "", ""},
 }
 
+func (it interpTest) Flags() map[string]bool {
```

Comment:
The fact that this map included both true and false items was a bit unintuitive to me at first. Perhaps add a doc comment like:

```suggestion
// Flags parses the "skip flags" in it.src's first comment and returns them as
// a map. An enable flag ("# +flag") appears as {"flag": true} in the map, and
// a disable flag ("# !flag") appears as {"flag": false} in the map.
func (it interpTest) Flags() map[string]bool {
```

Reply:
Ok